### PR TITLE
Increase AWS disk size

### PR DESCRIPTION
**Pull request by stgleb**

### CODE REVIEW

File: pkg/workflows/steps/amazon/create_machine.go [hunk 1]
```diff
@@ -99,7 +98,7 @@ func (s *StepCreateInstance) Run(ctx context.Context, w io.Writer, cfg *steps.Co
 	runInstanceInput := &ec2.RunInstancesInput{
 		BlockDeviceMappings: []*ec2.BlockDeviceMapping{
 			{
-				DeviceName: aws.String("/dev/xvda"),
+				DeviceName: aws.String("/dev/sda1"),
```
Comment (saheienko) [inline pkg/workflows/steps/amazon/create_machine.go]: Could it be set as a parameter?

File: pkg/clouds/aws/instances.go [hunk 2]
```diff
@@ -80,69 +80,6 @@ func (c *Client) AvailableInstanceTypes(ctx context.Context) ([]*EC2TypeInfo, er
 	return ec2Infos, nil
 }
 
-// CreateInstance starts a new instance due to the config.
-func (c *Client) CreateInstance(ctx context.Context, cfg InstanceConfig) (*ec2.Instance, error) {
```
Comment (saheienko) [inline pkg/clouds/aws/instances.go]: This is used by capacity.

Comment (stgleb) [reply pkg/clouds/aws/instances.go]: Okay, sure. I didn't know that.

File: pkg/workflows/steps/amazon/create_machine.go (hunk 1 above)
Comment (stgleb) [reply pkg/workflows/steps/amazon/create_machine.go]: I take root device name from AMI Image

File: pkg/clouds/aws/instances_test.go [hunk 3]
```diff
@@ -86,65 +86,6 @@ func TestNewClient(t *testing.T) {
 	}
 }
 
-func TestClient_CreateInstance(t *testing.T) {
```
Comment (saheienko) [inline pkg/clouds/aws/instances_test.go]: Could you keep tests too?

File: pkg/workflows/steps/config.go [hunk 4]
```diff
@@ -109,6 +109,7 @@ type AWSConfig struct {
 	MastersInstanceProfile string `json:"mastersInstanceProfile"`
 	NodesInstanceProfile   string `json:"nodesInstanceProfile"`
 	VolumeSize             string `json:"volumeSize"`
+	DeviceName 			   string `json:"deviceName"`
```
Comment (saheienko) [inline pkg/workflows/steps/config.go]: fmt

**Pull request merged**